configs/pie-components-config/vite.config.js

```diff
@@ -3,12 +3,6 @@ import dts from 'vite-plugin-dts';
 
 import { deepmerge } from 'deepmerge-ts';
 
-const bundledJetDeps = [
-    '@justeattakeaway/pie-components-config',
-    '@justeattakeaway/pie-css',
-    '@justeattakeaway/pie-webc-testing',
-];
```

Comment:
Removing these does not change the output bundles!

1. `pie-components-config` is only used at buildtime for things like vite config, TS config, test config, etc.
2. `pie-css` is not actually used inside the components, we ask consumers to add it to their applications.
3. `pie-webc-testing` is naturally only needed for tests.